gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/TestHelper.java

```diff
@@ -51,11 +52,19 @@
  * @author Stephen Mallette (http://stephen.genoprime.com)
  */
 public final class TestHelper {
+    private static final Logger logger = LoggerFactory.getLogger(TestHelper.class);
 
+    public static final Random RANDOM;
     private static final String SEP = File.separator;
     private static final char URL_SEP = '/';
     public static final String TEST_DATA_RELATIVE_DIR = "test-case-data";
 
+    static {
+        final long seed = Long.parseLong(System.getProperty("testSeed", String.valueOf(System.currentTimeMillis())));
+        logger.info("*** THE RANDOM TEST SEED IS {} ***", seed);
+        RANDOM = new Random();
```

Comment:
This should be `RANDOM = new Random(seed);`.

Reply:
wow - the whole point of the PR and i didn't even do it right :disappointed:  :rage4: